src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -666,15 +666,26 @@ def _handle_group_actions(
             and "text" in response
         ):
             del response["text"]
-        slack_client = SlackClient(integration_id=slack_request.integration.id)
 
         if not slack_request.data.get("response_url"):
             # XXX: when you click an option in a modal dropdown it submits the request even though "Submit" has not been clicked
             return self.respond()
-        try:
-            slack_client.post(slack_request.data["response_url"], data=response, json=True)
-        except ApiError as e:
-            logger.error("slack.action.response-error", extra={"error": str(e)})
+
+        response_url = slack_request.data["response_url"]
+        if not features.has("organizations:slack-sdk-webhook-handling", group.project.organization):
+            slack_client = SlackClient(integration_id=slack_request.integration.id)
+            try:
+                slack_client.post(response_url, data=response, json=True)
+            except ApiError as e:
+                logger.error("slack.action.response-error", extra={"error": str(e)})
```

Comment:
Do we want a metric here?

Reply:
i add metrics during feature flag cleanup, but yes